requirements.txt

```diff
@@ -32,6 +32,7 @@ selenium==3.141.0
 requests==2.25.0
 azure-storage-blob==12.4.0
 azure-core==1.8.0
+msrest==0.6.21
```

Comment:
Note: this is a development issue. We need to pin msrest to 0.6.21 to get it to work with our version of azure-core. (see [here](https://github.com/Azure/azure-sdk-for-python/issues/24765)). We might want to add this to a different PR, though.